cmd/ore/hetzner/create.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hetzner
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/flatcar/mantle/sdk"
+)
+
+var (
+	cmdCreate = &cobra.Command{
+		Use:   "create-image",
+		Short: "Create image on Hetzner",
+		Long: `Upload an image to Hetzner.
+
+After a successful run, the final line of output will be the ID of the image.
+`,
+		RunE: runCreate,
+	}
+
+	path  string
+	name  string
+	board string
+)
+
+func init() {
+	Hetzner.AddCommand(cmdCreate)
+	cmdCreate.Flags().StringVar(&path, "file",
+		sdk.BuildRoot()+"/images/amd64-usr/latest/coreos_production_openstack_image.img",
+		"Flatcar image (can be an absolute path or an URL)")
+	cmdCreate.Flags().StringVar(&name, "name", "", "image name")
+	cmdCreate.Flags().StringVar(&board, "board", "amd64-usr", "board of the image")
+}
+
+func runCreate(cmd *cobra.Command, args []string) error {
+	id, err := API.UploadImage(cmd.Context(), name, path, board)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Couldn't create image: %v\n", err)
+		os.Exit(1)
```

Comment:
Fixed in [ee307db](https://github.com/flatcar/mantle/pull/536/commits/ee307db7ecdca00e0a0e576785ab16420c96a3dd). I also updated the `preflightCheck` in `hetzner.go` to return an error instead of `os.Exit()`ing